tools/clang/unittests/HLSLExec/LongVectors.cpp

```diff
@@ -431,6 +450,62 @@ runTest(ID3D12Device *D3DDevice, bool VerboseLogging,
   return OutData;
 }
 
+// LoadAndStore operations dynamically configure sizes on the underlying
+// resources based on the vector size and data type size.
+void configureLoadAndStoreShaderOp(
+    const Operation &Operation, const DataType &OpDataType, size_t VectorSize,
+    size_t ElementSize, std::shared_ptr<st::ShaderOpSet> &ShaderOpSet) {
```

Comment:
It doesn't look like this function actually takes ownership of ShaderOpSet does it? I'm pretty sure it isn't going to reassign it to something else?

```suggestion
    size_t ElementSize, st::ShaderOpSet *ShaderOpSet) {
```